src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/FileResultExecutorBase.cs

```diff
@@ -41,6 +41,7 @@ internal enum PreconditionState
             ActionContext context,
             FileResult result,
             long? fileLength,
+            bool enableRangeProcessing,
```

Comment:
Is .Infrastructure considered public API? the class is public and the method is protected, so people could derive from it.

Reply:
It was .Internal until recently.

@Eilon is Infrastructure just a name change and is considered internal still?

Reply:
@jbagga was this in `.Infrastructure` in 2.0.0? That's the salient point

If this class was in `.Internal` then it has no support bar in 2.0.0 and there is no breaking change 😆

Reply:
Or rather, it's a change that we're willing to make

Reply:
It was .Internal in 2.0.0 https://github.com/aspnet/Mvc/blob/rel/2.0.0/src/Microsoft.AspNetCore.Mvc.Core/Internal/FileResultExecutorBase.cs#L16